src/chrome/content/rules/StartCom.xml

```diff
@@ -62,6 +56,6 @@ Fetch error: http://auth.startssl.com/ => https://auth.startssl.com/: (35, 'erro
 
   <!-- host startcom.org responds neither on 80 nor on 443,
     but should be protected from MitM and redirected to https://www.startcom.org -->
-  <rule from="^http://startcom\.org/" to="https://www.startcom.org/" />
```

Comment:
Yes, instead of excluding anything in auth\\\.startssl\\\.com/.*\\\.cr[lt], exclude \\\.cr\[lt\](?:$|\?) for everything, i.e.
```xml
<exclusion pattern="^http://.*\.cr[lt](?:$|\?)" />
```
and don't include auth.startssl.com among the tests required by the test coverage checker heuristics for that exclusion.